Drawer shadow fixes

davyd-akamai opened the pull request:
1. Dropdown component tokens update

- `dropdown.icon.default` has been renamed to `dropdown.icon.primary` **[major change]**
- Value of the `dropdown.background.selected` has been changed to `interaction.background.primaryActive`
- Value of the `dropdown.background.category` has been changed to `interaction.background.neutralHover`
- New `dropdown.icon.secondary` token
- New `dropdown.border.default` token

2. New tokens and tokens updates

- New `interaction.border.light` token for less prominent interactive borders
- New `neutral.85` token to expand a neutral palette
- Value of the `interaction.background.primaryHover` has been changed to `neutral.85`

3. Hot-fix for a drawer shadow tokens
4. Fixed a bug where the background of a disabled checkbox had the same color as the table row hover color

jaalah-akamai committed:
Merge pull request #102 from linode/staging

Release v5.2.0 - staging → main

davyd-akamai committed:
Drawer shadows fixes

davyd-akamai committed:
Dropdown component tokens update, new interaction.border token

jaalah-akamai commented:
@davyd-akamai is this something we want to move forward with? cc: @abailly-akamai just in case

davyd-akamai commented:
@jaalah-akamai @abailly-akamai Yes, I think we can merge it, there is no upcoming changes in the nearest future.

abailly-akamai commented:
@davyd-akamai should we wait for UIE-9838?

davyd-akamai commented:
> @davyd-akamai should we wait for UIE-9838?

Yes, good point. Let me fix that first.

davyd-akamai committed:
New 'Neutral.85' global token, update for an 'interaction.background.primaryHover' alias token

davyd-akamai commented:
@abailly-akamai @aaleksee I’ve introduced a new “Neutral.85” global token and changed the value of the interaction token responsible for the table row hover color. This should resolve the issue highlighted in UIE-9838. It seems our neutral palette should be expanded to cover all dark mode scenarios, and I’m working on that.

abailly-akamai reviewed (commented):
This looks good ty @davyd-akamai !
